src/test/java/com/iexec/core/task/update/TaskUpdateRequestManagerTests.java

```diff
@@ -105,4 +146,36 @@ public void shouldNotUpdateAtTheSameTime() throws NoSuchFieldException, IllegalA
             foundOutputsForKeyGroup.merge(updateId, 1, (currentValue, defaultValue) -> currentValue + 1);
         }
     }
+    // endregion
+
+    // region createQueue()
+    @Test
+    void shouldGetInOrder() {
```

Comment:
I think contribution deadline is missing in this test.

Reply:
I agree with this

Reply:
I agree with this

Reply:
Sure, thanks for pointing it out. Fixed in https://github.com/iExecBlockchainComputing/iexec-core/pull/479/commits/af0e77ceb3352628c76f91f1798c2f984abbbb2e.